cranelift-codegen/src/ir/function.rs

```diff
@@ -74,6 +74,13 @@ pub struct Function {
     /// Track the original source location for each instruction. The source locations are not
     /// interpreted by Cranelift, only preserved.
     pub srclocs: SourceLocs,
+
+    /// Frame layout for the instructions.
+    ///
+    /// The stack unwinding requires to have information about which registers and where they
+    /// are saved in the frame. This information is created during the prologue and epilogue
+    /// passes.
+    pub frame_layout: FrameLayout,
```

Comment:
Could this be an Option type, so it is generated and filled only when a flag is set?